Jenkinsfile

```diff
@@ -40,7 +40,7 @@ boolean release_candidate() {
 def scons_faults_args() {
     // The default build will have BUILD_TYPE=dev; fault injection enabled
     if ((cachedCommitPragma(pragma: 'faults-enabled', def_val: 'true') == 'true') && !release_candidate()) {
-        return "BUILD_TYPE=dev"
+        return "BUILD_TYPE=dev --debug=explain"
```

Comment:
what is this?

Reply:
This was there to try to help debugging on centos and leap 15 system.  It was not suppose to go into source control and has been eliminated.